docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -82,7 +82,55 @@ For all available options, refer to the https://playwright.dev/docs/test-configu
 Do not attempt to run in headful mode (using `headless:false`) when running through Elastic's global managed testing infrastructure or Private Locations as this is not supported.
 ====
 
-Below are details on a few Playwright options that are particularly relevant to Elastic Synthetics including timeouts, timezones, and device emulation.
+Below are details on a few Playwright options that are particularly relevant to Elastic Synthetics including TLS client authentication, timeouts, timezones, and device emulation.
+
+[discrete]
+[[synthetics-configuration-playwright-options-client-certificates]]
+== TLS client authentication
+To enable TLS client authentication, set https://playwright.dev/docs/api/class-testoptions#test-options-client-certificates[`clientCertificates`] option in the configuration file:
+
+//////
+[NOTE]
+=====
+Path-based options `{certPath, keyPath, pfxPath}` are only supported on Private Locations, defer to in-memory alternatives `{cert, key, pfx}` when running on locations hosted by Elastic.
+=====
+//////
```

Comment:
Are the notes supposed to be hidden? This note and the "tip" below are currently commented out (wrapped in `//////`).

Reply:
Oops, fixed that. I hadn't noticed when I copied the markdown from another section, thanks for pointing it out.

@colleenmcginnis would you happen to know why I can't see my changes in the preview URL? 
Preview url: https://observability-docs_bk_4351.docs-preview.app.elstc.co/guide/en/observability/current/synthetics-configuration.html

Reply:
When you're targeting `main`, you're updating the `master` path (https://observability-docs_bk_4351.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-configuration.html). You should be able to see your changes at that link. It can be added to `current` via a backport to `8.15` using the `backport-8.15` label.